docs/source/markdown/podman-machine-init.1.md

```diff
@@ -47,6 +47,10 @@ Defaults to `testing`.
 
 Memory (in MB).
 
+#### **--volume**, **-v**=*source:target*
+
+Mounts a volume from `source` to `target`.
```

Comment:
This should be a much richer explanation.  
Something like:
```
The source volume will become available to local podman commands.  If you mounted your home directory into the podman machine. Then `podman run  -v /home/dwalsh:/home/dwalsh ...` commands will be able to mount content from these machine directories into containers.
```

Reply:
You might want to comment on PR #11454 instead, where this commit was taken from